### PR TITLE
Included link to pair programming sessions

### PR DESCRIPTION
**Pull request by starlord-daniel**
Added links to pair programming hands-on sessions we've been running in multiple teams (either MS only or customer & MS joined teams) to increase use of pair programming, especially at the beginning of joint coding engagements.

### CODE REVIEW

**Review by TessFerrandez — APPROVED** [review]

File: agile-development/collaboration/readme.md [hunk 1]
```diff
@@ -69,3 +69,8 @@ As soon the pair finds out that the PBI will warrant swarming, the pair brings i
 ## Benefits of increased collaboration
 
 Knowledge sharing and bringing CSE and customer engineers together in a ‘code-with’ manner is an important aspect of CSE engagements. This grows both our customers’ and our CSE team’s capability to build on Azure. We are responsible for demonstrating engineering fundamentals and leaving the customer in a better place after we disengage. This can only happen if we collaborate and engage together as a team. In addition to improved software quality, this also adds a beneficial social aspect to the engagements.
+
+## Resources
+
```
Comment (meulta) [inline agile-development/collaboration/readme.md:74]: Looks good, can you please add a note saying that these are examples and can be adapted to each project? 
In the second link, there is a list of fixed time duration that might not fit all projects.

Thanks!

**Review by meulta — CHANGES_REQUESTED** [review]

File: agile-development/collaboration/readme.md (hunk 1 above)
Comment (starlord-daniel) [reply agile-development/collaboration/readme.md:74]: Sure, on it.

Comment (starlord-daniel) [reply agile-development/collaboration/readme.md:74]: Also, the time duration is a pattern which we tested with some teams (including customer and internal), but can be adapted as well, of course

Comment (starlord-daniel) [reply agile-development/collaboration/readme.md:74]: @meulta Is the change reflecting what you had in mind?

Comment (meulta) [reply agile-development/collaboration/readme.md:74]: Yep thanks!!

**Review by meulta — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**